resources/language/resource.language.en_gb/strings.po

```diff
@@ -180,3 +180,35 @@ msgstr ""
 msgctxt "#30042"
 msgid "Using a SOCKS proxy requires the PySocks library (script.module.pysocks) installed."
 msgstr "Using a SOCKS proxy requires the PySocks library (script.module.pysocks) installed."
+
+msgctxt "#30043"
+msgid "Extracting Widevine CDM"
+msgstr ""
+
+msgctxt "#30044"
+msgid "Prepare downloaded image..."
```

Comment:
For` xbmcgui.DialogProgress()` messages I think it's better to use the present participle of the verb: preparing

Reply:
Agreed.